python/dask_cudf/dask_cudf/backends.py

```diff
@@ -667,6 +667,41 @@ def from_dict(
             constructor=constructor,
         )
 
+    @classmethod
```

Comment:
nitpick: use `staticmethod` like in the other read methods?

```suggestion
    @staticmethod
```